packages/vite-plugin/test/plugin.test.ts

```diff
@@ -29,8 +31,8 @@ afterEach(async () => {
 
 describe('plugin', () => {
   it('with defaults', async () => {
-    const baseConfig = await resolveConfig({}, 'serve')
-    const resolvedConfig = await resolveConfig({ plugins: [CloudstackVitePlugin()] }, 'serve')
+    const baseConfig = await resolveConfig({}, 'build')
```

Comment:
check if it's not because of `build` that this test is outputing "can't find favicon.svg"